src/components/editor/editor-component.ts

```diff
@@ -141,16 +141,18 @@ export const actions = {
     const data = _state.textEditor.editor.getValue()
     await writeNote(_state.node.id, NoteDataType.TEXT, data)
     _actions.updateLastSaveTime()
+    _state.node.lastModified = new Date()
   },
 
   saveHandwritingNote: () => async (_state: any, _actions: any) => {
     _actions.handwritingEditor.component.getImage(async (data: any) => {
       await writeNote(_state.node.id, NoteDataType.HANDWRITING, data)
       _actions.updateLastSaveTime()
+      _state.node.lastModified = new Date()
     })
   },
 
-  updateLastSaveTime: () => () => ({
+  updateLastSaveTime: () => ({
```

Comment:
You should be careful here. I have not tested it myself, but there are two forms of actions. As long as this is doing the right thing, it's fine to use it.

```
const actions = {
  setValue: value => ({ value })
}

const actions = {
  down: value => state => ({ count: state.count - value }),
  up: value => state => ({ count: state.count + value })
}
```